api/Document.json

```diff
@@ -2521,6 +2521,7 @@
           "support": {
             "chrome": {
               "version_added": "1"
+              "notes": "From Chrome 110, <code>domain</code> will be behind a flag. See <a href='https://chromestatus.com/feature/5428079583297536'</a>."
```

Comment:
```suggestion
              "version_removed": "110",
              "notes": "From Chrome 110, <code>domain</code> is behind a flag. See this <a href='https://chromestatus.com/feature/5428079583297536'>chromestatus entry</a>."
```

Reply:
You also need a comma in the line above:

"version_added": "1" 
"version_added": "1",

Reply:
hm wait, it isn't removed entirely, is it? It will still be available by it opt-in via `Origin-keyed agent clusters`  https://origin-agent-cluster-demo.dev